src/Qdrant.Client/Grpc/Expression.cs

```diff
@@ -0,0 +1,126 @@
+namespace Qdrant.Client.Grpc
+{
+    /// <summary>
+    /// To construct a formula for score boosting.
+    /// </summary>
+    public partial class Expression
+    {
+        /// <summary>
+        /// Implicitly converts a float constant into an <see cref="Expression"/>.
+        /// </summary>
+        public static implicit operator Expression(float constant) =>
+            new() { Constant = constant };
+
+        /// <summary>
+        /// Implicitly converts a variable name (string) into an <see cref="Expression"/>.
+        /// Used to reference a payload key or a score variable.
+        /// </summary>
+        public static implicit operator Expression(string variable) =>
+            new() { Variable = variable };
```

Comment:
Given there are other `string -> Expression` methods (dateTime and dateTimeKey), it may be confusing to have an implicit conversion for this one. 

Note that there is no implicit conversion for any of the `Expression -> Expression` conversions, they are all static methods (neg, abs, etc.).

May want to expose this as a `FromVariable(...)` static method, unless you think that this will be the super common case and warrants being an implicit conversion?

Reply:
> unless you think that this will be the super common case and warrants being an implicit conversion?

Yes. See

https://github.com/qdrant/landing_page/blob/d82ea122b46f7b0f7debc6d22d6767484f38ae68/qdrant-landing/content/documentation/headless/snippets/query-points/score-boost-closer-to-user/csharp.md?plain=1#L20

and

https://github.com/qdrant/landing_page/blob/d82ea122b46f7b0f7debc6d22d6767484f38ae68/qdrant-landing/content/documentation/headless/snippets/query-points/score-boost-tags/csharp.md?plain=1#L20